pygfx/controllers/_orbit.py

```diff
@@ -69,8 +69,8 @@ def _update_rotate(self, delta):
         delta_azimuth, delta_elevation = delta
         camera_state = self._get_camera_state()
 
-        # Note: this code does not use la.vec_euclidian_to_spherical and
-        # la.vec_spherical_to_euclidian, because those functions currently
+        # Note: this code does not use la.vec_euclidean_to_spherical and
+        # la.vec_spherical_to_euclidean, because those functions currently
```

Comment:
This is actually spelled as the method is currently called (see https://github.com/pygfx/pylinalg/blob/main/pylinalg/vector.py#L386). If anything, we should update in pylinalg.

Reply:
https://github.com/pygfx/pylinalg/pull/84